warehouse/migrations/versions/a81f6cb8fc31_remove_malware_infrastructure.py

```diff
@@ -0,0 +1,184 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Remove malware infrastructure
+
+Revision ID: a81f6cb8fc31
+Revises: fd0479fed881
+Create Date: 2023-05-11 12:13:31.300595
+"""
+
+import citext
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "a81f6cb8fc31"
+down_revision = "fd0479fed881"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏either rebase alembic heads, or merge them with another migration:
```shell
docker compose run --rm web python -m warehouse db merge -m "merge c5f718 and a81f6c" c5f718 a81f6c
```
and add the new migration to commit scope.
Ref: https://alembic.sqlalchemy.org/en/latest/branches.html#merging-branches

Reply:
https://github.com/pypi/warehouse/pull/13647/commits/980cbabe27b728a1c1c9244047d3c19ba5f7004d